common/app/views/fragments/items/facia_cards/contentCard.scala.html

```diff
@@ -176,27 +177,29 @@
             }
 
             case Some(InlineSlideshow(imageElements)) => {
-                <div class="fc-item__media-wrapper">
-                    <div class="fc-item__image-container u-responsive-ratio fc-item__slideshow fc-item__slideshow--@imageElements.size">
-                        @imageElements.headOption.map { imageElement =>
-                            @captionedImage(
-                                classes = Seq("responsive-img "),
-                                widths = item.mediaWidthsByBreakpoint,
-                                maybePath = Some(imageElement.url),
-                                maybeSrc = if(containerIndex == 0 && index < 9) Some(imageElement.url) else None,
-                                caption = imageElement.caption
-                            )
-                            @imageElements.tail.map { imageElement =>
+                @defining(imageElements.take(if(ActiveExperiments.isParticipating(NoMoreThanFiveSlides)) 5 else 10)) { slides =>
+                    <div class="fc-item__media-wrapper">
+                        <div class="fc-item__image-container u-responsive-ratio fc-item__slideshow fc-item__slideshow--@slides.size">
+                            @slides.headOption.map { imageElement =>
                                 @captionedImage(
                                     classes = Seq("responsive-img "),
                                     widths = item.mediaWidthsByBreakpoint,
                                     maybePath = Some(imageElement.url),
+                                    maybeSrc = if(containerIndex == 0 && index < 10) Some(imageElement.url) else None,
```

Comment:
We are jumping from 9 to 10 here, is that intentional?

Reply:
```suggestion
                                    maybeSrc = if(containerIndex == 0 && index < 9) Some(imageElement.url) else None,
```